zerver/tests/test_email_mirror.py

```diff
@@ -1174,7 +1174,7 @@ def test_receive_email_response_for_auth_failures(self) -> None:
 
         self.assertEqual(
             message.content,
-            "Error sending message to stream announce via message notification email reply:\nOnly organization administrators can send to this stream.",
+            "Error sending message to stream announce via message notification email reply:\nOnly organization administrators can send to this channel.",
```

Comment:
`stream announce` should be updated?

Reply:
Yes, missed this as it's not marked for translation. I assume that should also be corrected.

Reply:
Updated to now be "Error sending message to channel {channel_name} via message ..." and marked for translation.